### ISSUE TITLE
remove vendor prefixes

### ISSUE THREAD

**Issue by VinSpee** [mattn/emmet-vim#284]
Hi!

May I suggest that we remove vendor prefixes from CSS snippets? This should be handled in a build step and creates a point of maintenance for you. Also, it's hard to get `flexbox` expansion right. Maybe at least make it configurable.

**Comment by jwhitley** [comment]
Agreed.  Using a tool like [Autoprefixer](https://github.com/postcss/autoprefixer) keeps the source CSS clean, readable, and easier to maintain.  It's also trivial to add or remove prefixes (e.g. to support a specific browser version) by changing Autoprefixer's build parameters.

**Comment by mattn** [comment]
Do you expect to add new option to disable vendor prefixes?

**Comment by VinSpee** [comment]
I would expect them to be disabled by default, but if we have to provide an option that would be acceptable to me. 



(null)

**Comment by mattn** [comment]
For example, which the part of functionality of emmet do you think? expand abbreviation?

**Comment by VinSpee** [comment]
Sorry, I don't understand what you're asking. Could you clarify it?

**Comment by mattn** [comment]
Ah, i'm asking just simple.
Do you expect removing https://github.com/mattn/emmet-vim/blob/master/autoload/emmet.vim#L1565-L1569 ?

**Comment by VinSpee** [comment]
That's an interesting question. I think you should remove all prefixes, but I could see the argument for keeping expansions that are not "standard". see postcss/autoprefixer#322 for their reasoning, which I think is very smart.

**Comment by VinSpee** [comment]
also https://github.com/mattn/emmet-vim/blob/master/autoload/emmet.vim#L977 should not be prefixed
